command/alias.go

```diff
@@ -168,11 +174,10 @@ func aliasList(cmd *cobra.Command, args []string) error {
 }
 
 var aliasDeleteCmd = &cobra.Command{
-	Use:     "delete <alias>",
-	Short:   "Delete an alias.",
-	Args:    cobra.ExactArgs(1),
-	Example: "gh alias delete co",
```

Comment:
I didn't believe this example added anything over the usage line already displayed: `gh alias delete <alias>`